aws/v16.4.2/README.md

```diff
@@ -0,0 +1,14 @@
+# :zap: Giant Swarm Release v16.4.2 for AWS :zap:
+
+<< Add description here >>
```

Comment:
@kubasobon, @gianfranco-l we need a description for the release in order to make it available to our customers. Could you provide one, given that your fix is the only change in this release? Thanks!

Reply:
@alex-dabija created related [pr here](https://github.com/giantswarm/releases/pull/989/files) cc @ljakimczuk  @piontec @MarcelMue

Reply:
Thanks! I tweaked it a bit and added the release announcement.